src/helm/proxy/clients/anthropic_client.py

```diff
@@ -18,13 +19,145 @@
 from .client import Client, wrap_request_time, truncate_sequence
 
 
+class AnthropicClient(Client):
+    """
+    Client for the Anthropic models (https://arxiv.org/abs/2204.05862).
+    They use their own version of the GPT-2 tokenizer.
```

Comment:
Could you confirm with the Anthropic folks that they're still using the GPT-2 tokenizer?

Reply:
I've just asked, and I am waiting for their response.

Reply:
So I checked with Anthropic and apparently they are using their own tokenizer and not gpt2. Should I change this or do we want to keep it like this for now? Do you think using gpt2 instead of their tokenizer could be the cause of this completion? TheĠU.S.ĠambassadorĠtoĠRussia,ĠJohnĠBeyrle,ĠsaidĠtheĠvideoĠclipsĠwereĠfabricated.ĠAccordingĠtoĠtheĠpassage:ĊĊ"BeyrleĠwasĠunequivocalĠinĠexpressingĠherĠsupportĠinĠanĠinterviewĠwithĠABCĠNews.ĠĠĊĊ

Reply:
Changed to their tokenizer. It solved this issue also.